src/watertap_contrib/reflo/analysis/case_studies/KBHDP/components/MD.py

```diff
@@ -281,7 +281,7 @@ def get_concentrate_flow(b):
         return b.concentrate.properties[0].flow_vol_phase["Liq"] == pyunits.convert(
             (
                 b.unit.mp.get_active_process_blocks()[-1].fs.vagmd.system_capacity
```

Comment:
Just double checking on this. Is system capacity the volume of water (total feed) being treated or the volume of treated water produced (total permeate)?

Reply:
Ah thanks for spotting that, you're definitely correct (capacity = total permeate).
I just committed the correct equation, so that` concentrate_volume = system_capacity * (1-recovery) / recovery`